samples/tlssurvey/src/main/kotlin/okhttp3/survey/ssllabs/SslLabsScrape.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2016 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3.survey.ssllabs
+
+import com.squareup.moshi.Moshi
+import okhttp3.Call
+import okhttp3.OkHttpClient
+import okhttp3.survey.ssllabs.SslLabsService
+import okhttp3.survey.types.Client
+import okhttp3.survey.types.SuiteId
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+
+class SslLabsScraper(
```

Comment:
I slightly object to the word _scraper_ here, as it suggests we’re doing something untoward?

The SSL labs people are awesome and offer their data under a generous creative commons license!
https://www.ssllabs.com/downloads/Qualys_SSL_Labs_Terms_of_Use.pdf

Lemme send a PR to rename this!

Reply:
https://github.com/square/okhttp/pull/8324